content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -115,38 +162,147 @@ Similarly, you must pass the corresponding public key to the `kube-apiserver`
 using the `--service-account-key-file` flag. The public key will be used to
 verify the tokens during authentication.
 
-#### To create additional API tokens
+### ServiceAccount admission controller
 
-A controller loop ensures a Secret with an API token exists for each
-ServiceAccount. To create additional API tokens for a ServiceAccount, create a
+The modification of pods is implemented via a plugin
+called an [Admission Controller](/docs/reference/access-authn-authz/admission-controllers/).
+It is part of the API server.
+This admission controller acts synchronously to modify pods as they are created or
+updated. When this plugin is active (and it is by default on most distributions), then
+it does the following when a Pod is created or modified:
+
+1. If the pod does not have a `.spec.serviceAccountName` set, the admission controller sets the name of the
+   ServiceAccount for this incoming Pod to `default`.
+1. The admission controller ensures that the ServiceAccount referenced by the incoming Pod exists. If there
+   is no ServiceAccount with a matching name, the admission controller rejects the incoming Pod. That check
+   applies even for the `default` ServiceAccount.
+1. Provided that neither the ServiceAccount's `automountServiceAccountToken` field nor the
+   Pod's `automountServiceAccountToken` field is set to `false`:
+   - the admission controller mutates the incoming Pod, adding an extra
+     {{< glossary_tooltip text="volume" term_id="volume" >}} that contains
+     a token for API access.
+   - the admission controller adds a `volumeSource` to each container in the Pod.
+     For Linux containers, that `volumeSource` is mounted at `/var/run/secrets/kubernetes.io/serviceaccount`
+   - If the spec of the incoming Pod does already contain any `imagePullSecrets`, then the
+     admission controller adds `imagePullSecrets`, copying them from the `ServiceAccount`.
+
+### TokenRequest API
```

Comment:
Would it be worth mentioning the new `kubectl create token` feature here as that uses the TokenRequest API and could be used by end-users to generate new tokens?

Reply:
Maybe. Is that a “makes the page better” change, or a “page doesn't make sense without it”. If it's the former, I'd prefer to move this one forward towards merge, and also track an issue about also documenting `kubectl create token` - maybe with a new task page.

Reply:
How's this new wording?